package/dropbear/0001-Makefile.in-changes.patch.conditional

```diff
@@ -0,0 +1,76 @@
+--- a/Makefile.in	2016-07-21 20:47:09.000000000 +0530
```

Comment:
Patch needs appropriate format-patch header, including commit log providing a reason for the patch. For more info see: https://git-scm.com/docs/git-format-patch

Reply:
Yes, made the corrections and now created the patch with git format-patch header.